pyproject.toml

```diff
@@ -71,13 +71,15 @@ dev-no-pandas = [
     "jupyter",
     "quartodoc>=0.11.0; python_version >= '3.9'",
     "griffe==0.38.1",
+    "plotnine",
```

Comment:
I think the last piece is putting plotnine in dev (line 64), since it induces a pandas dependency. The dev-no-pandas is used to run our pandas free tests.